### PR TITLE
CMake update after Eyescale/CMake#341

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -33,7 +33,14 @@ set(COMMON_PROJECT_DOMAIN eu.humanbrainproject)
 
 include(Common)
 
-include(FindPackages)
+if(LINUX)
+  common_package(avahi-client)
+else()
```
Comment (hernando) [inline CMakeLists.txt]: Is DNSSD available in Windows? Otherwise use elseif(APPLE) as in the original code.

File: servus/CMakeLists.txt [hunk 2]
```diff
@@ -17,7 +17,6 @@
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
 set(SERVUS_PUBLIC_HEADERS
-  ${COMMON_INCLUDES}
```
Comment (hernando) [inline servus/CMakeLists.txt]: I didn't quite get this in the CMAKE PR. These files are now added automatically to the sources by common_library?

File: CMakeLists.txt (hunk 1 above)
Comment (tribal-tec) [inline CMakeLists.txt]: Yes, the official Apple BonjourSDK is available on Windows.

File: servus/CMakeLists.txt (hunk 2 above)
Comment (tribal-tec) [inline servus/CMakeLists.txt]: Yes

**Pull request merged**